src/main/java/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package strategy;
+
+import java.util.Map;
+import model.FruitTransaction;
+import service.OperationHandler;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
Ensure that the class is focused on a single responsibility. This class seems well-designed in terms of the SOLID principles, as it delegates the handling of operations to respective handlers.